src/command_modules/azure-cli-dla/azure/cli/command_modules/dla/_help.py

```diff
@@ -25,6 +25,11 @@
 helps['dla account blob-storage add'] = """
 type: command
 short-summary: Links an Azure Storage account to the specified Data Lake Analytics account.
+examples:
+  - name: Links an Azure Storage account to the specified Data Lake Analytics account. (autogenerated)
+    text: |-
+        az dla account blob-storage add --access-key "{Azure Storage Account Key}" --account "{Data Lake Analytics account name}" --storage-account-name MyStorageAccount
+    crafted: true
```

Comment:
I wasn't aware of this. Hopefully, down the line we can incorporate this into the documentation output we send to the docs team.

Reply:
We add crafted field to all the Aladdin generated examples we add.

Reply:
Sounds good!